coralproject/talk
# [CORL-1649] Bugfix: Comment draft deleted on login or registration

## What does this PR do?

- creates new hooks wrapping `usePersistedState` allowing use of either local or session storage
- ~switch `PostCommentForm:draft` from session to local storage so it isn't removed when session is cleared via login/signup~
- Prevent sessionStorage from being cleared after login



## What changes to the GraphQL/Database Schema does this PR introduce?
none


## How do I test this PR?

1. logout
2. type text into the input box
3. click "register" and register a new user
4. ensure text is still in input box
